com.unity.formats.alembic/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 ### Changed
 ### Fixed
+- Fixed bug that caused previous Alembic instance in prefabs to get converted to out of project references.
```

Comment:
Suggestion:
`- Fixed a bug that was converting previous Alembic instance in prefabs into out-of-project references.`